### DISCUSSION TITLE
Project not available

### DISCUSSION THREAD

**Discussion by alphandlouis** [vercel/community#3549] [Help]
### Summary

Hello, Why my project are not displayed while locally they work (see photo)

Can you explain to me ?

Cordially

### Example

https://vercel.com/alphandlouis/react-scoot-multi-page-website / https://react-scoot-multi-page-website.vercel.app/

https://vercel.com/alphandlouis/react-todo-app-main / https://react-todo-app-main-seven.vercel.app/

### Steps to Reproduce
![Screenshot_1](https://github.com/vercel/community/assets/100416219/d824251a-4ae9-4670-85be-033466938956)

**Comment by amyegan** [comment]
Hey there! If you share your public repo or a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), then someone here may recognize what is going wrong.

**Reply by alphandlouis** [reply]
Oops sorry, here are 2 examples that show you my problem

https://vercel.com/alphandlouis/react-scoot-multi-page-website / https://react-scoot-multi-page-website.vercel.app/

https://vercel.com/alphandlouis/react-todo-app-main / https://react-todo-app-main-seven.vercel.app/

Cordially
